tests/functional/test_endpoint_rulesets.py

```diff
@@ -189,7 +183,7 @@ def iter_e2e_test_cases_that_produce(endpoints=False, errors=False):
                     expected_endpoint,
                     marks=pytest.mark.skipif(
                         'sigv4a' in expected_authschemes,
-                        reason="Test case expects sigv4a which required CRT",
+                        reason="Test case expects sigv4a which requires CRT",
                     ),
```

Comment:
I just noticed this. I'd expect the conditional to be something closer to `'sigv4a' in expected_authschemes and not HAS_CRT`. Are these cases being run elsewhere or are we just dropping them?

Reply:
That's an error, thank you for catching that. Fixed in 8188e27. This affected one test case. I confirmed that there are no other uses of `skipif` left that get this wrong.